inputs_demand.md

```diff
@@ -0,0 +1,172 @@
+
+#### Share weight interpolation rules
+
+Share weight interpolation rules are specified in [A32.subsector_shrwt.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A32.subsector_shrwt.csv), [A32.subsector_interp.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A32.subsector_interp.csv), [A321.subsector_shrwt.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A321.subsector_shrwt.csv), [A321.subsector_interp.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A321.subsector_interp.csv), [A322.subsector_shrwt.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A322.subsector_shrwt.csv), [A322.subsector_interp.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A322.subsector_interp.csv), [A44.subsector_shrwt.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A44.subsector_shrwt.csv), [A44.subsector_interp.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A44.subsector_interp.csv), [A54.globaltranTech_shrwt_revised.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A54.globaltranTech_shrwt_revised.csv), [A54.globaltranTech_interp_revised.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A54.globaltranTech_interp_revised.csv), [A54.tranSubsector_shrwt_revised.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A54.tranSubsector_shrwt_revised.csv), [A54.tranSubsector_interp_revised.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A54.tranSubsector_interp_revised.csv). For each sector, the file that ends `_interp` specifies the rule (e.g., fixed, linear) and the file that ends `_shrwt` indicates the value to interpolate to (if needed).
+
+##### Fuel preference elasticities
+
+Fuel preference elasticities are specified in [A32.fuelprefElasticity.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A32.fuelprefElasticity.csv), [A44.fuelprefElasticity.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A42.fuelprefElasticity.csv).
+
+##### Value of time in transit
+
+Multipliers used to determine the value of time in transit are specified in [A54.tranSubsector_VOTT_revised.csv](https://github.com/JGCRI/gcam-core/blob/master/input/gcamdata/inst/extdata/energy/A54.tranSubsector_VOTT_revised.csv).
+
+### Water
+
+
+
+
+
+#### Description
+
+| Name | Description | Type | Source | Resolution | Unit |
```

Comment:
| Livestock water coefficients | Water coefficients for drinking and the servicing of livestock commodities, includes data for the period 1996-2005 | Mekonnen, M. M., & Hoekstra, A. Y. (2010). The green, blue and grey water footprint of farm animals and animal products. Volume 2: Appendices | livestock type | liters per head per day |
| Electricity cooling system shares | Histroical shares of cooling system types associated with power plants aggregated to GCAM3 regions | UCS and Schakel Inventories | GCAM3 region, power plant type, cooling system type, water type (fresh, seawater), and year | Unitless |
| Electricity water coefficients | Water withdrawal and consumption coefficients for power plants and cooling system types | Macknick et al., 2011 | fuel, power plant type, cooling system type, water type (fresh, seawater) | $$m^3$$ per MWh |
| Primary energy water coefficients | Water coefficients for the consumption of water during the process of mining primary energy fuel sources | Maheu, A. (2009). Energy choices and their impacts on demand for water resources: An assessment of current and projected water consumption in global energy production. Unisféra. | global, fuel, mining technology, water type (consumption) | $$m^3$$ per TJ |
| Municipal water withdrawals | Water withdrawal values for municipalities include data, as reported, from 1987 to 2017 | [FAO Aquastat](#fao2016) | GCAM region, year | $$km^3$$ |
| Municipal water use efficiency | Water efficiency values for municipalities | [Shiklomanov 2000](#shiklomanov2000) | Continent | Percent |
| Municipal water cost | Price per unit of water delivered to municipalities | International Benchmarking Network for Water and Sanitation Utilities ([IBNET](#ibnet)) | Country | USD per $$km^3$$ |

Reply:
please note the citations are not in proper format